packages/components/src/Image/tests/chromatic/SvgImage.stories.tsx

```diff
@@ -0,0 +1,52 @@
+
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Inline } from "../../../layout/index.ts";
+import { NoResults } from "../../assets/index.ts";
+import { SvgImage } from "../../src/SvgImage.tsx";
+
+const meta = {
+    title: "Components/SvgImage",
+    component: SvgImage,
+    args: {
+        src: NoResults,
+        "aria-label": "No Results"
+    }
+} satisfies Meta<typeof SvgImage>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Stroke = {
+    render: args => (
```

Comment:
export const Stroke = {
    args:{
        stroke: "neutral"
    }
} satisfies Story;

Reply:
you don't need render function everywhere